### DISCUSSION TITLE
ValueError: not enough values to unpack (expected 2, got 1)

### DISCUSSION THREAD

**Discussion by michail-p** [microsoft/FLAML#229] [Q&A]
Fit() throws this error for trainX. Any idea how to fix that? my array is like this [[x, y, z], [a, b, c]]

**Comment by sonichi** [comment]
Could you show the code snippet and the trace?

**Reply by michail-p** [reply]
Fixed it, had to use dataframe instead of numpy array for X_train

**Reply by sonichi** [reply]
numpy array is supported for X_train. For example, see https://github.com/microsoft/FLAML/blob/main/notebook/automl_in_sklearn_pipeline.ipynb
